packages/manager/modules/pci/src/projects/project/billing/legacy/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cpbc_billing_control": "Facturation",
+  "cpbc_billing_control": "Billing control",
```

Comment:
**question:** If content is not meant to be translated, couldn't we use it directly as a constant?